addons/pinniped/config-controller/controllers/controllers.go

```diff
@@ -73,6 +73,10 @@ func (c *PinnipedController) SetupWithManager(manager ctrl.Manager) error {
 	return nil
 }
 
+// +kubebuilder:rbac:groups="",resources=secrets,verbs=list;watch;get;patch;update;delete
+// +kubebuilder:rbac:groups="",resources=configmaps,verbs=watch;get
```

Comment:
can we add the `list` verb here, I think you were right when you called out that we might need it.

Reply:
aha yes!